src/components/TodoErrors/TodoErrors.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useState, useContext, useEffect } from 'react';
+import cn from 'classnames';
+import { TodosContext } from '../../TodosContext';
+
+export const TodoErrors: React.FC = () => {
+  const {
+    errorMessage,
+    setErrorMessage,
+  } = useContext(TodosContext);
+
+  const [isHidden, setIsHidden] = useState(false);
+
+  useEffect(() => {
+    const timerId = setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+
+    return () => clearTimeout(timerId);
+  }, []);
+
+  return (
+    <div
+      className={
+        cn(
+          'notification',
+          'is-danger',
+          'is-light',
+          'has-text-weight-normal',
+          { hidde: isHidden },
+        )
+      }
+    >
+      <button
+        type="button"
+        className="delete"
+        onClick={() => setIsHidden(true)}
```

Comment:
the error is not hidden when clicking on the cross
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/54936044/e97ae981-64ab-4e9e-8ce6-d7fd9451e74d)